vars/maven.groovy

```diff
@@ -132,9 +135,16 @@ def uploadLocalArtifacts(String mvnUploadCredsId, String artifactDir, String rep
     }
 }
 
+def getLatestArtifactFromRepository(String repositoryUrl, String groupId, String artifactId) {
```

Comment:
I would name it `getLatestArtifactVersionFromRepository`

Reply:
done